airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -58,8 +58,8 @@ def download_catalog(catalog_url):
 METADATA_FILE_NAME = "metadata.yaml"
 ICON_FILE_NAME = "icon.svg"
 
-IMPORTANT_CONNECTOR_THRESHOLDS = {
-    "sl": 300,
+STRATEGIC_CONNECTOR_THRESHOLDS = {
+    "sl": 200,
```

Comment:
Im worried about this lowering 😬 

It seems like we may inadvertendly cause a lot of connectors to start failings tests

can you confirm this is correct?

Reply:
I believe these thresholds are only used in the context of required reviewers so there should be no impact on the tests.